example/timer/odp_timer_accuracy.c

```diff
@@ -516,7 +526,7 @@ static int start_timers(test_global_t *test_global)
 	odp_queue_param_init(&queue_param);
 	queue_param.type        = ODP_QUEUE_TYPE_SCHED;
 	queue_param.sched.prio  = odp_schedule_default_prio();
-	queue_param.sched.sync  = ODP_SCHED_SYNC_PARALLEL;
+	queue_param.sched.sync  = test_global->opt.queue_type;
```

Comment:
SCHED_SYNC_XXX values are implementation specific. So, need to map user input to a value: 0->PARALLEL, etc